modules/Classroom/hooks/useExamsApiCall.ts

```diff
@@ -0,0 +1,44 @@
+import { showNotification } from "@mantine/notifications";
+
+import { NOTIFICATION_AUTO_CLOSE_TIMEOUT_IN_MILLISECONDS } from "@/shared/constants/app.constants";
+import { useCreateExamMutation } from "@/shared/redux/rtk-apis/exams/exams.api";
+import { TCreateExamDto } from "@/shared/redux/rtk-apis/exams/exams.types";
+import { parseApiErrorMessage } from "@/shared/utils/errors";
+
+import { TCreateExamFormData } from "../components/ClassroomExamModal/ClassroomExamModal.types";
+
+const useExamsApiCall = (classroomId: number) => {
```

Comment:
The naming is very confusing. Please rename it to something like useCreateExamForm


```suggestion
export const useCreateExamForm = (classroomId: number) => {
```